File: python/7_kyu/case_swapping.py
# https://www.codewars.com/kata/5590961e6620c0825000008f/train/python

def swap(string):
    return "".join(list(map(lambda x: x.lower() if x == x.upper() else x.upper(), string)))

# Not good for production but it's fun to solve these katas with a one liner hehe

File: CodeWars/javascript/7_kyu/case_swapping.js
// https://www.codewars.com/kata/5590961e6620c0825000008f/train/python

function swap(str) {
  return str.split("").map((c) => {
    return c == c.toLowerCase() ? c.toUpperCase() : c.toLowerCase();
  }).join("");
}
